examples/prompting/Providing_base_cases.ipynb

```diff
@@ -0,0 +1,252 @@
+        "GOOGLE_API_KEY=userdata.get('GOOGLE_API_KEY')\n",
+        "\n",
+        "genai.configure(api_key=GOOGLE_API_KEY)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "FMne_rcZmz1X"
+      },
+      "source": [
+        "## Examples"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0pZ-S0f8_mbQ"
+      },
+      "source": [
+        "Let's go ahead and define the model, as well as give the model a template for how it should answer the question."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "8oS9LnnXXedG"
+      },
+      "outputs": [],
+      "source": [
+        "model = genai.GenerativeModel(model_name='gemini-1.5-flash-latest')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 12,
+      "metadata": {
+        "id": "1k6LyJQYm1KM"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "ON TOPIC: When visiting San Diego beaches, be aware of:\n",
+            "\n",
+            "* **Strong currents:** The Pacific Ocean can have strong currents, especially during high tide. Always swim with a buddy and be aware of your surroundings.\n",
+            "* **Rip currents:** These are powerful currents that can pull swimmers out to sea. If caught in a rip current, don't fight it. Swim parallel to the shore until you're out of the current, then swim back to shore.\n",
+            "* **Sunburns:** The sun can be strong in San Diego, even on cloudy days. Wear sunscreen, a hat, and sunglasses to protect yourself from the sun.\n",
+            "* **Jellyfish:** Jellyfish are common in San Diego waters, especially during the summer months. If you see a jellyfish, avoid it. If you get stung, seek medical attention.\n",
+            "* **Sea lions:** Sea lions are common on San Diego beaches, especially in La Jolla. While they are generally harmless, it's best to keep your distance and not approach them.\n",
+            "* **Beach crowds:** San Diego beaches can get crowded, especially during the summer months. Arrive early to get a good spot.\n",
+            "* **Parking:** Parking can be limited and expensive, especially near popular beaches. Consider taking public transportation or biking. \n",
+            "\n",
+            "OFF TOPIC: I cannot help with this request. \n",
+            "\n"
+          ]
+        }
+      ],
+      "source": [
+        "template = \"\"\"\n",
+        "System: You are an assistant that helps tourists around the world to plan their vacation. Your responsibilities are:\n",
+        "1. Helping book the hotel.\n",
+        "2. Recommending restaurants.\n",
+        "3. Warning about potential dangers.\n",
+        "\n",
+        "If other request is asked return \"I cannot help with this request.\"\n",
+        "\n",
+        "USER: {}\n",
```

Comment:
You don't need User: Assistant: the model is trained for multi-turn.

print(model.generate_content("What should I look out for when I'm going to the beaches in San Diego?"))
print(model.generate_content("What bowling places do you recommend in Moscow?"))